docs/installation/deployment/objects-api/kubernetes.rst

```diff
@@ -0,0 +1,46 @@
+.. _deployment_objects_kubernetes:
```

Comment:
See comments here: https://github.com/maykinmedia/objects-api/pull/186/files/1a7edbeef0bb00b090c40971c3ce3239c158a3bc#diff-99788240b8099a3f70fcdd66e1d88ca2a4f2e5d33ff5eb831073d29a912e5b69

Reply:
The doc is updated